pkg/proxy/endpoint.go

```diff
@@ -84,18 +86,28 @@ func (c *EndpointClient) SupportsEventType(connect bool, eventType string) bool
 	return false
 }
 
+// SupportsContext takes the context string of an event, and determines whether the endpoint supports
+// this context
+func (c *EndpointClient) SupportsContext(context string) bool {
+	if c.connect {
+		return context != ""
+	}
```

Comment:
What's `context` in relation to `connect` here?

Reply:
this is still in flight from api-events but afaik `context` will be the acct ID of the connected account for connect events in v2